package.json

```diff
@@ -1,78 +1,92 @@
 {
-  "name": "rate-limit-memcached",
-  "description": "Memcached client for the express-rate-limit middleware.",
-  "version": "1.0.0",
-  "author": "linyows",
-  "license": "MIT",
-  "repository": "linyows/rate-limit-memcached",
-  "homepage": "https://github.com/linyows/rate-limit-memcached",
-  "keywords": [
-    "express",
-    "api",
-    "memcached",
-    "rate-limit"
-  ],
-  "bugs": "https://github.com/linyows/rate-limit-memcached/issues",
-  "main": "./index.js",
-  "types": "./lib/memcached-store.d.ts",
-  "files": [
-    "MIT-LICENSE",
-    "README.md",
-    "index.js",
-    "lib/"
-  ],
-  "directories": {
-    "lib": "./lib",
-    "src": "./src"
-  },
-  "dependencies": {
-    "memcached": "^2.2.2"
-  },
-  "devDependencies": {
-    "@types/memcached": "^2.2.4",
-    "@types/node": "^10.12.18",
-    "ava": "^1.4.1",
-    "codecov": "^3.2.0",
-    "nyc": "^14.1.0",
-    "testdouble": "^3.9.2",
-    "ts-node": "^7.0.0",
-    "tslint": "^5.10.0",
-    "tslint-microsoft-contrib": "^6.0.0",
-    "typescript": "~2.8.3"
-  },
-  "engines": {
-    "node": ">=8.9.0"
-  },
-  "ava": {
-    "compileEnhancements": false,
-    "extensions": [
-      "ts"
-    ],
-    "require": [
-      "ts-node/register"
-    ],
-    "files": [
-      "src/**/*.test.ts"
-    ]
-  },
-  "nyc": {
-    "extension": [
-      ".ts"
-    ],
-    "exclude": [
-      "src/**/*.test.ts"
-    ]
-  },
-  "scripts": {
-    "build": "tsc",
-    "clean": "rm -rf lib",
-    "ci": "npm i && npm run lint && npm run cov && npm audit",
-    "test": "ava -v",
-    "cov": "nyc ava",
-    "cov:report": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "lint": "tslint -p tsconfig.json -c tslint.json src/**/*.ts",
-    "preversion": "npm run clean && npm run build && npm test",
-    "postversion": "git push origin --tags",
-    "prepublishOnly": "npm run clean && npm run build && npm test"
-  }
+	"name": "rate-limit-memcached",
+	"version": "1.0.0",
+	"description": "A memcached store for the express-rate-limit middleware.",
+	"author": "linyows",
+	"license": "MIT",
+	"homepage": "https://github.com/express-rate-limit/rate-limit-memcached",
+	"repository": "express-rate-limit/rate-limit-memcached",
+	"keywords": [
+		"express",
+		"api",
+		"memcached",
+		"rate-limit"
+	],
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./dist/index.d.mts",
+				"default": "./dist/index.mjs"
+			},
+			"require": {
+				"types": "./dist/index.d.cts",
+				"default": "./dist/index.cjs"
+			}
```

Comment:
What do you think about dropping this and making rate-limit-memcached v2 an ESM-only module?

Might be a good place to test it out before doing the same with express-rate-limit.

Reply:
I think it's a good idea, let's do it!